SECURITY.md

```diff
@@ -1,26 +1,13 @@
 # Security Policy
 
-Dark Reader is known to have no intentional security vulnerabilities.
-If reported, vulnerabilities will be fixed in the shortest time possible,
-with minimal impact possible.
+Any security vulnerabilities discovered in Dark Reader will be fixed as soon as possible with as minimal impact as possible when reported.
 
-## Supported Versions
+Please ensure that you are always running the [latest version available](https://github.com/darkreader/darkreader/releases).
 
-Latest version is supported.
+Some web browser extension stores may have an older version currently published.
 
-| Version   | Supported          |
-| --------- | ------------------ |
-| 4.9.43    | :white_check_mark: |
-| < 4.9.43  | :x:                |
+## Security Vulnerability Report
 
-## Reporting a Vulnerability
+If you have discovered a security vulnerability, please email us at <darkreaderapp@gmail.com> immediately.
 
-In case there is a vulnerability, please immediately notify
-<darkreaderapp@gmail.com>.
-
-If you can fix it yourself, please also open a pull request at
-<https://github.com/darkreader/darkreader/pulls>.
-
-Alexander Shutau, owner  
-"Gusted" William Zijl, maintainer  
-Erkin Alp Güney, security policy author  
+If you can fix the security vulnerability, please open a [pull request](https://github.com/darkreader/darkreader/pulls).
```

Comment:
I prefer if we avoid this. As nobody can guarantee it will be merged and published ASAP.